x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/empty_prompt/empty_prompt.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { EuiButton, EuiCallOut, EuiEmptyPrompt, EuiImage, EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import dfaImage from './data_frame_analytics_kibana_image.png';
+import { useMlKibana } from '../../../../../contexts/kibana';
+
+interface Props {
+  disabled: boolean;
+  onCreateFirstJobClick: () => void;
+}
+
+export const AnalyticsEmptyPrompt: FC<Props> = ({ disabled, onCreateFirstJobClick }) => {
+  const {
+    services: {
+      http: { basePath },
+    },
+  } = useMlKibana();
+  const transformsLink = `${basePath.get()}/app/management/data/transform`;
+  return (
+    <EuiEmptyPrompt
```

Comment:
I think it would be nice to add a link to the DFA docs at the bottom of this prompt, similar to what's done in Data view management. It could link to the [overview docs page](https://www.elastic.co/guide/en/machine-learning/current/ml-dfa-overview.html) perhaps?

![image](https://user-images.githubusercontent.com/7405507/150777089-57d8540a-752b-4ece-ac72-a6b78415b6e7.png)

Also what do you think of the styling used in the Data views prompt - with the filled background and no border?

Reply:
Updated styles in https://github.com/elastic/kibana/pull/123462/commits/c378bdf73f241cce60cbdea44ce4bd0252ebc0a6 and added a footer with a docs link (to this page https://www.elastic.co/guide/en/machine-learning/master/ml-dfanalytics.html) in https://github.com/elastic/kibana/pull/123462/commits/fc1712071818082de2fa295b33e0fd342e70a732